lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -2178,9 +2196,9 @@ LogicalResult StmtEmitter::visitStmt(InstanceOp op) {
   // name, then use it here.  This is a hack because we lack proper support for
   // parameterized modules in the RTL dialect.
   if (auto extMod = dyn_cast<RTLModuleExternOp>(moduleOp)) {
-    indent() << extMod.getVerilogModuleName();
+    indent() << emitter.getModuleName(extMod.getVerilogModuleName());
```

Comment:
You can get the StringAttr by using `extMod.verilogName()` IIRC.  This is a helper that digs the StringRef out of the StringAttr for convenience.

Reply:
Getting the ```StringAttr```, name was a bit non-trivial. 
For External modules, need to get the ```"sym_name"``` attribute if ```"verilogname``` does not exist, since it is optional. 
Also added the corresponding wrapper function for ```RTLModuleOp```, to get the ```StringAttr```